### ISSUE TITLE
Support images

### ISSUE THREAD

**Issue by leoskye** [aakashns/react-native-dialogs#30]
Any plans to add support for images in the dialog?

**Comment by birat-bade** [comment]
Yes I also need a library that supports image in the dialog box. Is there any update on this.

**Comment by Noitidart** [comment]
This library supports images. You just need to use html:

https://github.com/aakashns/react-native-dialogs/issues/78
